src/schemas/secrets.ts

```diff
@@ -0,0 +1,12 @@
+// Libraries
+import {schema} from 'normalizr'
+
+// Types
+import {ResourceType} from 'src/types'
+
+/* Tasks */
+
+// Defines the schema for the tasks resource
```

Comment:
What're the task references for?

Reply:
. . . leftover copypasta that should've been cleaned up.  Removed!